_config.template.yml

```diff
@@ -133,6 +133,10 @@ sidebar:
             #link: "/about"
             #icon: person
             #divider: false
+        timeline: #can use different languages
```

Comment:
I don't think we needn't add this configuration to config file. As you can see, we have gallery page and links page and they are not included in config file.
We needn't configuring it for users.